SU-SWS/stanford_profile
# Add and configure search api module to replace core search.

# READY FOR REVIEW

# Summary
- Use search api to replace core search functionality

# Need Review By (Date)
- no rush

# Urgency
- low

# Steps to Test
1. `composer require su-sws/stanford_profile:dev-search-api --no-update`
2. `composer update -n`
3. `drush cim -y; drush cr; drush cron`
4. try the site search
5. verify it returns some results that you'd expect and is has some decent formatting to it.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
